docs/architecture/anti-entropy/hinted-handoff.rst

```diff
@@ -43,4 +43,65 @@ Hinted handoff is enabled and managed by these settings in :code:`scylla.yaml`:
 
 Storing of the hint can also fail. Enabling hinted handoff therefore does not eliminate the need for repair; a user must recurrently :doc:`run a full repair </operating-scylla/procedures/maintenance/repair/>` to ensure data consistency across the cluster nodes.
 
+Sync point HTTP API
+^^^^^^^^^^^^^^^^^^^
+
+ScyllaDB provides you with a means to wait until hints for given nodes have been replayed. It's realized by creating a so-called sync point that corresponds to the hints that have accumulated up to the time of its creation. It's always created for both regular hints and updates for materialized views. You can create and await sync points by sending HTTP requests to :code:`http://{API_ADDRESS}:{API_PORT}/hinted_handoff/sync_point/`, where :code:`{API_ADDRESS}` corresponds to the API IP address of the coordinator node which hints are supposed to be sent from.
+
+You can create a sync point by sending a POST request to :code:`/hinted_handoff/sync_point` with the following parameters:
+
+* :code:`target_hosts`: a list of the IP addresses of nodes towards which hints should be replayed; the IPs are divided by commas, e.g. :code:`127.0.0.2,127.0.0.3`. If no IP address is specified, ScyllaDB assumes the sync point should be created towards all live nodes in the cluster. This parameter is not required -- if it's not present, ScyllaDB will treat the request as if zero IP addresses were specified.
```

Comment:
> :code:`target_hosts`: a list of the IP addresses of nodes towards which hints should be replayed

It's a bit vague in my opinion, this could suggest that creating a sync point waits for something. I'd rephrase it, say something about "including" hints targeted to those specific hosts. To make your point clear, you can also say what will awaiting a sync point created for a given `target_hosts` will do.